### DISCUSSION TITLE
API client

### DISCUSSION THREAD

**Discussion by nwithan8** [Dusk-Labs/dim#411] [Show and tell]
Haven't finished mapping all the API endpoints yet, but I have a rough Python library for Dim's API up and working: https://github.com/nwithan8/dim

**Comment by vgarleanu** [comment]
> Haven't finished mapping all the API endpoints yet, but I have a rough Python library for Dim's API up and working: https://github.com/nwithan8/dim

This looks awesome. A lot of the APIs are fairly stable but some will change in the near future. Mainly the rematching and the provider search apis.
